plenum/server/domain_req_handler.py

```diff
@@ -0,0 +1,118 @@
+import json
+
+from ledger.serializers.json_serializer import JsonSerializer
+from ledger.util import F
+from plenum.common.constants import TXN_TYPE, NYM, ROLE, STEWARD, TARGET_NYM, VERKEY
+from plenum.common.exceptions import UnauthorizedClientRequest
+from plenum.common.request import Request
+from plenum.common.txn_util import reqToTxn
+from plenum.common.types import f
+from plenum.persistence.util import txnsWithSeqNo
+from plenum.server.req_handler import RequestHandler
+from stp_core.common.log import getlogger
+
+logger = getlogger()
+
+
+class DomainRequestHandler(RequestHandler):
+    stateSerializer = JsonSerializer()
+
+    def __init__(self, ledger, state, reqProcessors):
+        super().__init__(ledger, state)
+        self.reqProcessors = reqProcessors
+
+    def validate(self, req: Request, config=None):
+        if req.operation.get(TXN_TYPE) == NYM:
```

Comment:
What about other types of requests in addition to NYM? Do we have request handlers for them?